test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.*;
+
+import static org.junit.Assert.assertTrue;
+
+public class SortCommandTest {
+
+    private AddressBook addressBook;
+
+    @Before
+    public void setUp() throws Exception {
+        Person C = new Person(new Name("CCC"), new Phone("61234567", false),
```

Comment:
A little nitpick here, doesn't really matter than much, but try to have an order when setting up -- i.e ABCD, then just call CADB later.